packages/fxa-admin-server/src/gql/account/account.resolver.ts

```diff
@@ -71,6 +78,22 @@ export class AccountResolver {
       .limit(10);
   }
 
+  // unverifies the user's email. will have to verify again on next login
+  @Mutation((returns) => Boolean)
+  public async unverifyEmail(
```

Comment:
we should add a test for this. there's one for a mutation in email-bounce you can reference https://github.com/mozilla/fxa/blob/main/packages/fxa-admin-server/src/gql/email-bounce/email-bounce.resolver.spec.ts#L79